kw

```diff
@@ -1,32 +1,35 @@
 #!/bin/bash
 
-# Note: the following line should not be modified, and neither the token should
-# be repeated in this file because It will be replaced during installation with
-# the respective installation prefix.
-KW_INSTALL_PREFIX=${KW_INSTALL_PREFIX:-"##KW_INSTALL_PREFIX_TOKEN##"}
-
 KWORKFLOW=${KWORKFLOW:-"kw"}
 
+# Global paths
+KW_LIB_DIR="${KW_LIB_DIR:-"$HOME/.local/lib"}/$KWORKFLOW"
+KW_SHARE="${XDG_DATA_HOME:-"$HOME/.local/share"}/$KWORKFLOW"
+KW_SHARE_DOC_DIR="$KW_SHARE_DIR/doc"
+KW_SHARE_MAN_DIR="$KW_SHARE_DIR/man"
+KW_SHARE_SOUND_DIR="$KW_SHARE_DIR/sound"
```

Comment:
Just for making this part of the code easier to read, could you add a new line between different sections? Maybe add a comment as well.